src/content/docs/ai-monitoring/configure-ai-monitoring.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Configure AI monitoring'
+metaDescription: 'You can apply certain configurations to your APM agents to change how your AI data appears in New Relic.'
+freshnessValidatedDate: never
+---
+
+Once you install AI monitoring, you can configure the default behavior of the agent or update your app to collect different kinds of data. 
+
+## Configure the agent [#configure-agents]
+
+Update default agent behavior for AI monitoring at these agent configuration docs: 
+
+* [Go](/docs/apm/agents/go-agent/configuration/go-agent-configuration/#ai-monitoring)
+* [Node.js](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#ai-monitoring)
+* [Python](/docs/apm/agents/python-agent/configuration/python-agent-configuration/#ai-monitoring)
+* [Ruby](/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/#ai-monitoring)
+
+## Enable token count API [#enable-token]
+
+If you've disabled `ai_monitoring.record_content.enabled`, you may still want to know the number of tokens your app uses in an interaction. If you haven't disabled content from being recorded by the agent, you don't need to invoke the token count API. 
```

Comment:
I would recommend updating the first sentence a bit. 

> If you've disabled `ai_monitoring.record_content.enabled`, it will prevent the agent sending the content to New Relic. In this scenario, if the token counts for each message will not be available in AI Monitoring. If you still want to know the number of tokens your app uses in an interaction, you will want to implement a callback and determine the token counts locally. If you haven't disabled content from being recorded by the agent, you don't need to implement the token count callback API.